docs/index.rst

```diff
@@ -35,6 +35,7 @@ Features
    configuration
    userGuide
    useAzkaban
+   userManager
    eventTrigger
```

Comment:
Shouldn't this userManager exist above useAzkaban?

Reply:
Good catch! WIll fix